src/new-client/src/Web/context.tsx

```diff
@@ -0,0 +1,4 @@
+import { createContext } from "react"
+export const TornOutTileContext = createContext({ isTornOut: false })
```

Comment:
This context doesn't belong in the Web folder as it is used outside of the web scope.

Reply:
Done, I didn't know where to put it so for now it's at `@/components/tearOutContext`  (I also changed the file name since it was misleading)